### PR TITLE
Removes computer idle & airlock 'failure-to-close' sounds

### PR DESCRIPTION
**Pull request by nauticall**
Soul crushing. Removes the idle sound from computers (that annoying buzzing white noise that plays) and the beeping noise that plays when standing in an airlock for too long. Oh the humanity!

Sounds themselves are still in the files, in case they want to be reimplemented for some other malicious reason.

### CODE REVIEW

**Comment by nauticall** [comment]
!review

**Comment by Greenjoe12345** [comment]
IMO maybe the airlock failure to close sound could be kept, but make the time between them quite a bit longer?

File: code/datums/looping_sounds/machinery_sounds.dm [hunk 1]
```diff
@@ -82,7 +82,6 @@
 	start_sound = 'sound/machines/computer/computer_start.ogg'
 	start_length = 7.2 SECONDS
 	start_volume = 10
-	mid_sounds = list('sound/machines/computer/computer_mid1.ogg' = 1, 'sound/machines/computer/computer_mid2.ogg' = 1)
 	mid_length = 1.8 SECONDS
```
Comment (NonQueueingMatt) [inline code/datums/looping_sounds/machinery_sounds.dm:85]: this needs to go too, and so do the sounds if you're removing their only usage

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -1945,9 +1945,6 @@ About the new airlock wires panel:
 		for(var/turf/turf in locs)
 			for(var/atom/movable/AM in turf)
 				if(AM.blocks_airlock())
-					if(world.time > next_beep_at)
-						playsound(src.loc, close_failure_blocked, 30, 0, -3)
-						next_beep_at = world.time + SecondsToTicks(10)
```
Comment (SleepyGemmy) [inline code/game/machinery/doors/airlock.dm:1950]: why? this is only used for secure airlocks.

**Comment by alsoandanswer** [comment]
I think these sounds are fine, dont need to remove them

**Comment by atteria** [comment]
> I think these sounds are fine, dont need to remove them

imo, both sounds are annoying and deserve, at minimum, replacement with tweaks to how they're played. I think removal is also a good option.

**Comment by DreamySkrell** [comment]
I don't care about computer sounds, but.

I think airlock sounds should stay. They are supposed to be annoying - like the whole point of airlocks is to keep air in, and they can't do that if they are open. We are on a ship, so the sound is a reminder to keep the airlocks closed. I'm fine with the delay or whatever being changed though.

**Comment by nauticall** [comment]
changes addressed. readded the airlock failure-to-close but made it take 20(ish) seconds to beep rather than 10, doubling it

**Review by NonQueueingMatt — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**